nibabel/filebasedimages.py

```diff
@@ -538,7 +540,30 @@ class SerializableImage(FileBasedImage):
     """
 
     @classmethod
-    def from_bytes(klass, bytestring):
+    def _filemap_from_iobase(klass, ioobject: io.IOBase):
```

Comment:
Consider `io_obj` instead of `ioobject`?   Just because it's a bit easier to read, and might be easier for non-English-speakers to parse.